### PR TITLE
Update on FAQ with webhook bitpay Questions

### LINKED ISSUE #852: Documenting Notification & Webhook events
Although users can refer to the Bitpay documentation for notification events, it is very strange this is not documented at all in the BTCPay Server docs. IPN questions are very common in the chat. 

We should (at the very least) mention the distinction between notification events and webhook events. 

Discussion: [#2282](https://github.com/btcpayserver/btcpayserver/discussions/2282)
Related: #774

### PR DESCRIPTION
**Pull request by Bangalisch**
Yooo

Looked at the open issue of bk #852 once more. 
Added the part of the discussion in https://github.com/btcpayserver/btcpayserver/discussions/2282 to FAQ 
Maybe needs better title ? otherwise i think this closes it and moves on in #1142  

closes #852

### CODE REVIEW

File: docs/FAQ/General.md [hunk 1]
```diff
@@ -158,6 +158,20 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
 
 ![Create new Webhook](../img/FAQ/btcpayWebhookFAQ2.jpg)
 
+## Webhook format is not bitpay compatible?
+
+Webhooks are not meant to be compatible with bitpay API. 
+There's two seperate IPNs in BTCPay Server.
+
+* Webhooks 
+* notifications
```
Comment (petzsch) [inline docs/FAQ/General.md:167]:
```suggestion
* notifications (In BitPay terms: "Instant Payment Notifications")
```

File: docs/FAQ/General.md [hunk 2]
```diff
@@ -158,6 +158,20 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
...
+
+Where Webhooks are the GreenField Events and Notifications are Bitpay events.
+Use `Notification URL` when creating an invoice through Bitpay.
+
+To read more on this question ; [Source](https://github.com/btcpayserver/btcpayserver/discussions/2282)
+
+To read more on [GreenField API ](https://docs.btcpayserver.org/API/Greenfield/v1/)
```
Comment (petzsch) [inline docs/FAQ/General.md:175]:
```suggestion

For guidance howto process a webhook in PHP, please check out the following [example script](https://github.com/btcpayserver/btcpayserver-greenfield-php/blob/master/examples/webhook.php).

```

**Review by petzsch — CHANGES_REQUESTED** [review]
2 minor suggestions:

1. Add the IPN term for users coming from a BitPay integration to know what we are talking about
2. add my favourite PHP example for webhooks here ^^

File: docs/FAQ/General.md [hunk 3]
```diff
@@ -158,6 +158,22 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
...
 
+## Webhook format is not bitpay compatible?
+
+Webhooks are not meant to be compatible with bitpay API. 
+There's two seperate IPNs in BTCPay Server.
+
+* Webhooks 
+* notifications (In BitPay terms: "Instant Payment Notifications")
+
+Where Webhooks are the GreenField Events and Notifications are Bitpay events.
```
Comment (ndeet) [inline docs/FAQ/General.md:169]: Nitpick: never saw Greenfield API to be camel cased like here so maybe better to keep it like we have it also on the [API docs](https://docs.btcpayserver.org/API/Greenfield/v1/)?

File: docs/FAQ/General.md [hunk 4]
```diff
@@ -158,6 +158,22 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
 
 ![Create new Webhook](../img/FAQ/btcpayWebhookFAQ2.jpg)
 
+## Webhook format is not bitpay compatible?
+
+Webhooks are not meant to be compatible with bitpay API. 
+There's two seperate IPNs in BTCPay Server.
```
Comment (ndeet) [inline docs/FAQ/General.md:164]: nitpick: you explain the abbreviation of IPN on line 167 below, but maybe you want to do that already here at its first occurrence and keep only the abbreviation on line 167 below

File: docs/FAQ/General.md (hunk 3 above)
Comment (Bangalisch) [reply docs/FAQ/General.md]: ack

**Review by ndeet — APPROVED** [review]
Thank you 💚

**Review by petzsch — APPROVED** [review]
👍

**Pull request merged**